loopdb/migration_03_last_hop.go

```diff
@@ -0,0 +1,86 @@
+package loopdb
+
+import (
+	"bytes"
+	"errors"
+	"fmt"
+
+	"github.com/btcsuite/btcd/chaincfg"
+	"github.com/coreos/bbolt"
+)
+
+// migrateLastHop migrates the database to v03, replacing the never used loop in
+// channel by a last hop pubkey.
+func migrateLastHop(tx *bbolt.Tx, chainParams *chaincfg.Params) error {
```

Comment:
nice no-dep migration :)

Reply:
Yes, definitely took care to make it no dep. We are wiser now